tests/suites/test_suite_bignum_mod_raw.function

```diff
@@ -294,7 +294,81 @@ exit:
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+/* BEGIN_CASE */
+void mpi_mod_raw_to_mont_rep( char * input_N, char * input_A, char * input_X )
+{
+    mbedtls_mpi N, A, X;
+    mbedtls_mpi_mod_modulus m;
+
+    mbedtls_mpi_init( &N );
+    mbedtls_mpi_init( &A );
+    mbedtls_mpi_init( &X );
+
+    /* Read inputs */
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &N, input_N ) );
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &A, input_A ) );
+    TEST_EQUAL( 0, mbedtls_test_read_mpi( &X, input_X ) );
```

Comment:
Should we be moving away from `mbedtls_test_read_mpi()` to using `mbedtls_test_read_mpi_core()` for tests that don't need an actual `mbedtls_mpi` but that really need a `mbedtls_mpi_uint *`?

Reply:
This is a simple test case so that makes sense. Patch added